### ISSUE TITLE
feature: show error message first as expected output in quickstart

### ISSUE THREAD

**Issue by aslom** [kubestellar/kubestellar#535]
### Feature Description

I missed initially that `NOTE: if you receive the error: 'curl: (52) Empty reply from server', wait and attempt curl again. It takes some time for the Apache HTTP Server to synchronize and start.` as it is after HTML successful output and I though quickstart failed because I missed a stop or did something wrong.

### Proposed Solution

Add output to quickstart:

```
curl http://localhost:8094
curl: (52) Empty reply from server
```

as it is expected output and then say tha need to wait few minutes to see

```
curl http://localhost:8096
<!DOCTYPE html>
<html>
  <body>
    This is a common web site.
  </body>
</html>
```

I would also add some commands to show status of Apache pods starting and their status.

### Want to contribute?

- [ ] I would like to work on this issue.

**Comment by clubanderson** [comment]
@dumb0002, @waltforme, or @francostellari do either of you have time to work on this in the 'main' branch version?  For the order of curl error, I do not have any suggestions.  For the pod status - maybe a 'watch' command for the pod status?

**Comment by aslom** [comment]
@clubanderson I will submit PR

**Comment by aslom** [comment]
@clubanderson https://github.com/kcp-dev/edge-mc/pull/541

**Comment by clubanderson** [comment]
fixed with #541
